app/townsquare/views.py

```diff
@@ -345,3 +361,12 @@ def offer_new(request):
         'nav': 'home',
     }
     return TemplateResponse(request, 'townsquare/new.html', context)
+
+def activity_score(activity):
+    if activity.activity_type in ACTIVITY_TYPE_SCORES:
+        base_score = settings.EDGERANK_SETTINGS['activity_weights'][ACTIVITY_TYPE_SCORES[activity.activity_type]]
```

Comment:
So even -ve activities like stop work etc would recieve a +ve score ?

Reply:
I think yes, even being something bad if people is doing to much to get tending, we will then need to understand what is happening